api/src/Page/Sample.php

```diff
@@ -355,8 +355,8 @@ function _add_simple_sample()
                 $isCapillary = sizeof($crystals) > 1 ? true : false;
 
```

Comment:
Since the last element in the `foreach` is persisted outside the scope of the loop due to how PHP scoping works, it would be clearer to explicitly set `$protein`, so that it's clearer for developers in the future

```suggestion

                $protein = $phases[0];
                
```